### PR TITLE
Initial skeleton for patch script

### PR DESCRIPTION
**Pull request by silentsakky**

### CODE REVIEW

File: patch/zcs-patch-8.7.11_GA_P1_1855/README [hunk 1]
```diff
@@ -0,0 +1,55 @@
+This patch includes following fixes on ZCS 8.7.11 GA
+ZCS-4115 - Bug 107925 - Persistent XSS - snippet [CWE-79]
+ZCS-4116 - Bug 108265 - Persistent XSS - message view as text [CWE-79]
+
+Please re-check your ZCS version. This patch should only be applied on 8.7.11 GA.
+In case of any issues after patch, you can revert the changes by ./install_patch -r
+
+-------------  To Install the Patch -------------
+
+root@zqa-036:/# tar -xvzf zcs-patch-8.7.11_P1_GA_1855.tgz
```
Comment (Prashantsurana) [inline patch/zcs-patch-8.7.11_GA_P1_1855/README]: what is 1855?   
the naming convention 
zcs-patch-8.7.11_GA_P1_XXXXX

Comment (silentsakky) [reply patch/zcs-patch-8.7.11_GA_P1_1855/README]: 8.7.11 GA is having 1854 so incremented it by one

**Comment by jiteshsojitra** [comment]
So are we going to add direct row files (modified) which is applicable for patch? Is there any way to pull that? Just my two cents if makes sense, ignore if not possible.

**Comment by silentsakky** [comment]
For now we don't have a patch system in place for 8.7.11, so this is temporary solution
in 8.8 we can implement a robust patch process for future use

**Comment by silentsakky** [comment]
Merged in f003ef77b2f135a384daccd61b120d5ce2c08121

**Pull request closed without merging**